soapboxd/application.go

```diff
@@ -39,6 +40,7 @@ func (s *server) CreateApplication(ctx context.Context, app *pb.Application) (*p
 
 	model := &models.Application{
 		ID:                 int(app.Id),
+		UserID:             sql.NullInt64{Int64: int64(app.UserId), Valid: true},
```

Comment:
If you define the column `user_id` to be `not null`, then you can just use a regular `int` here (`int(app.UserId)`)